src/lmql/models/lmtp/lmtp_replicate_client.py

```diff
@@ -0,0 +1,144 @@
+"""
+This file contains a client for tunneling LMTP over Replicate's protocol.
+"""
+
+import aiohttp
+import aiohttp_sse_client.client
+import asyncio
+import json
+import os
+import sys
+import warnings
+from .errors import LMTPStreamError
+
+class LMTPReplicateClient:
+    """
+    Simple client for tunneling LMTP into Replicate (https://replicate.com/)
+    """
+    def __init__(self, model_identifier, session, endpoint, **kwargs):
+        if 'REPLICATE_API_TOKEN' in os.environ:
+            self.api_key = os.environ['REPLICATE_API_TOKEN']
+        else: # FIXME: Allow API key to be passed in kwargs?
+            raise Exception('Must have REPLICATE_API_TOKEN defined to use Replicate transport')
```

Comment:
Maybe we can change this something like `Please provide a "REPLICATE_API_TOKEN" as environment variable to use the Replicate models.`, to make clear this is about an environment variable.